integration/templates/chart_operator_values.go

```diff
@@ -5,7 +5,7 @@ package templates
 // ChartOperatorValues values required by chart-operator-chart.
 const ChartOperatorValues = `
 clusterDNSIP: 10.96.0.10
-e2e: false
+e2e: true
```

Comment:
Yes we need this so the priority class is created for the integration tests. For Helm 2 EnsureTillerInstalled does this.

That priority class is so annoying!